<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tripSummary</class>
 <widget class="QDialog" name="tripSummary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>591</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Summary</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tripOrder_tableView">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>261</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="tripOrder_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Trip Order</string>
   </property>
  </widget>
  <widget class="QComboBox" name="school_comboBox">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>211</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="souvenirs_label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>170</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Souvenirs</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectSchool_label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select a Stadium</string>
   </property>
  </widget>
  <widget class="QTableView" name="souvenirsBySchool_tableView">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>261</y>
     <width>431</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="totalDistance_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>620</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Total Distance :</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalDistanceDisplay_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>620</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="schoolTotalDisplay_label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>591</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="schoolTotal_label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>590</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Stadium Total :</string>
   </property>
  </widget>
  <widget class="QLabel" name="tripTotalDisplay_label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>621</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="tripTotal_label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>621</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Trip Total :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
